README.md

```diff
@@ -90,19 +90,19 @@ If you do not wish to install the compiler globally with `-g` flag, you will nee
 See full usage documentation by typing:
 
 ```
-cadl --help
+typespec --help
 ```
 
-### Compiling Cadl source to an OpenAPI 3.0 specification
+### Compiling TypeSpec source to an OpenAPI 3.0 specification
 
-Here is a very small Cadl example that uses the `@cadl-lang/openapi3` library to generate OpenAPI 3.0 from Cadl.
+Here is a very small TypeSpec example that uses the `@typespec/openapi3` library to generate OpenAPI 3.0 from TypeSpec.
 
-#### sample.cadl
+#### sample.typespec
 
-```cadl
-import "@cadl-lang/rest";
+```typespec
```

Comment:
These are usually short too, I wonder if this should be ` ```tsp `

Reply:
@bterlson, can you guys make a call? tsp may be short, would it be too short and cryptic to users?

Reply:
According to the [GitHub docs](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting), the names that are identified for fenced code blocks can be found in this file:

https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

I believe they match based on the language name (the key of the entry like "TypeScript") and also any aliases that are defined for the language.  I think we could use both `typespec` and `tsp`!  Luckily neither of those are taken :)

Reply:
fixed.